File: tests/fixtures/fixture_ts/src/services/orderService.ts
import { processPayment, refundPayment } from '@lib/payment';
import { Order, calculateTotal } from '@models/order';
import { cacheGet, cacheSet } from '@lib/cache';

export function placeOrder(items: number[]): { orderId: string; paid: boolean } {
  const total = calculateTotal(items);
  const order: Order = { id: crypto.randomUUID(), userId: 'u1', total };
  const result = processPayment(order);
  cacheSet(`order:${order.id}`, order);
  return { orderId: order.id, paid: result.success };
}

export function cancelOrder(orderId: string, transactionId: string): boolean {
  cacheGet(`order:${orderId}`);
  return refundPayment(transactionId);
}

File: tests/fixtures/fixture_python/app/services/order_service.py
from ..models.order import Order, calculate_total
from ..utils.cache import cache_get, cache_set

def place_order(items: list) -> dict:
    total = calculate_total(items)
    order = Order(id='o1', user_id='u1', total=total)
    cache_set(f'order:{order.id}', order)
    return {'order_id': order.id, 'total': total}

def cancel_order(order_id: str) -> bool:
    cached = cache_get(f'order:{order_id}')
    return cached is not None
